.github/workflows/release.yml

```diff
@@ -11,9 +15,11 @@ jobs:
         with:
           fetch-depth: 0
       - uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adoptium@11
```

Comment:
Will we still realease on 11?

Reply:
> Will we still realease on 11?

Ouch, missed that. Fixed to 8 now.

Reply:
> Maybe it makes more sense to build on the same jvm as test but put the `-target` option to go to 1.8?

We are testing on both 8 and 17 now. BTW, they are run in parallel, so it doesn't add time to check the PR. (It does increase the probability of flaky tests failing though).